config.json

```diff
@@ -74,6 +74,12 @@
       ]
     },
     {
+      "slug": "twelve-days",
+      "difficulty": 2,
```

Comment:
I think this is probably slightly more difficult than a 2. Maybe you could bump this up a bit, and also add a topic or two? I'm having a hell of a time thinking of good topics for this, though... If I think of anything, I'll come back with them!